library/ssl_tls13_client.c

```diff
@@ -404,11 +405,121 @@ static int ssl_tls13_write_client_hello( mbedtls_ssl_context *ssl )
     return ret;
 }
 
+/*
+ * Handler for MBEDTLS_SSL_SERVER_HELLO
+ */
+static int ssl_tls1_3_read_server_hello( mbedtls_ssl_context *ssl )
+{
+    MBEDTLS_SSL_DEBUG_MSG( 1, ( "%s hasn't been implemented", __func__ ) );
+    mbedtls_ssl_handshake_set_state( ssl, MBEDTLS_SSL_ENCRYPTED_EXTENSIONS );
+    return( 0 );
```

Comment:
```suggestion
    return( MBEDTLS_ERR_SSL_FEATURE_UNAVAILABLE );
```
Rather return in error I'd say, ditto below.

Reply:
Not return `MBEDTLS_ERR_SSL_FEATURE_UNAVAILABLE ` is for run all states . That can make sure the state machine work correctly.

Reply:
Only `ssl_tls1_3_handshake_wrapup` will return `MBEDTLS_ERR_SSL_FEATURE_UNAVAILABLE`

Reply:
I was also initially inclined to return an error here, but I can see how it makes sense to return 0 instead. Anyway, this is temporary, to I don't think it's worth spending too much time on it, so I'm OK with returning 0.